python-azure-img-utils.spec

```diff
@@ -15,52 +15,59 @@
 # Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
-
-Name:           python3-azure-img-utils
+%{?sle15_python_module_pythons}
+%global skip_python2 1
```

Comment:
We also need the alternative setup, i.e. 
```
%if 0%{?suse_version} > 1500
%bcond_without libalternatives
%else
%bcond_with libalternatives
%endif
```
In Factory this will be built for multiple versions and therefore the command needs to be set via alternatives.